custom_components/meural/config_flow.py

```diff
@@ -23,11 +24,10 @@ async def validate_input(hass: core.HomeAssistant, data):
     """
     session = hass.helpers.aiohttp_client.async_get_clientsession()
     try:
-        token = await pymeural.authenticate(session, data["email"], data["password"])
+        return await pymeural.authenticate(session, data["email"], data["password"])
```

Comment:
Can we detect if the authentication is invalid? Does it retunr a specific status code that we can detect on `aiohttp.ClientError`?

Reply:
done